ui/app/templates/components/secret-version-menu.hbs

```diff
@@ -0,0 +1,96 @@
+    <BasicDropdown 
+      @class="popup-menu" 
+      @horizontalPosition="auto-right" 
+      @verticalPosition="below" 
+      as |D|
+    >
+      <D.trigger 
+        data-test-popup-menu-trigger="true"
+        @class={{concat "popup-menu-trigger button is-ghost has-text-grey" (if D.isOpen " is-active")}} 
+        @tagName="button" 
+      >
+        {{#if useDefaultTrigger}}
+          <ICon aria-label="More options" @glyph="more" @size="16" @class="has-text-black auto-width" />
```

Comment:
ah, funny. is the component named `ICon` because ember forces each component to have a dash in the name? 🤣

Reply:
Lol exactly - the new angle bracket syntax doesn't have that limitation, but we'll have to convert it everywhere at once.